_stbt/core.py

```diff
@@ -412,20 +412,20 @@ class MatchResult(object):
 
     * ``time`` (float): The time at which the video-frame was captured in
       seconds since 1970-01-01T00:00Z.  This timestamp can be compared with
-      system time (`time.time()`).  Introduced in v26.
+      system time (`time.time()`).
     * ``match``: Boolean result, the same as evaluating `MatchResult` as a bool.
       That is, ``if match_result:`` will behave the same as
       ``if match_result.match:``.
     * ``region``: The `Region` in the video frame where the image was found.
     * ``first_pass_result``: Value between 0 (poor) and 1.0 (excellent match)
       from the first pass of stb-tester's two-pass image matching algorithm
       (see `MatchParameters` for details).
-    * ``frame``: The video frame that was searched, in OpenCV format.
+    * ``frame`` (`Frame`): The video frame that was searched.
```

Comment:
`frame` is not necessarily a `Frame` if the user has passed in a frame they've loaded themselves.